### PR TITLE
Feat/live 8375 fw update redesign

### PR DESCRIPTION
**Pull request by ofreyssinet-ledger**
### 📝 Description

Requires the flag `newFwUpdateUxFeatureFlag` to be enabled.

### ❓ Context

- **Impacted projects**: `llm` 
- **Linked resource(s)**: [LIVE-8375] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://github.com/LedgerHQ/ledger-live/assets/91890529/8789ad4c-fd40-4369-bb17-ae22f56ae42b

IMG_0067

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-8375]: https://ledgerhq.atlassian.net/browse/LIVE-8375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/theme/InvertTheme.tsx
Comment (ofreyssinet-ledger) [inline apps/ledger-live-mobile/src/components/theme/InvertTheme.tsx:1]: unused

Comment (alexandremgo) [reply apps/ledger-live-mobile/src/components/theme/InvertTheme.tsx:1]: 🧹

**Review by alexandremgo — APPROVED** [review]
LGTM !

File: apps/ledger-live-mobile/src/components/theme/InvertTheme.tsx
Comment (ofreyssinet-ledger) [reply apps/ledger-live-mobile/src/components/theme/InvertTheme.tsx:1]: 💅

File: apps/ledger-live-mobile/src/components/FirmwareUpdateBanner.tsx [hunk 1]
```diff
@@ -179,50 +179,57 @@ const FirmwareUpdateBanner = ({ onBackFromUpdate }: FirmwareUpdateBannerProps) =
...
     ? getDeviceModel(lastConnectedDevice.modelId).productName
-    : "";
+    : undefined;
+
+  const deviceName = lastConnectedDevice?.deviceName;
+
+  /**
+   * TODO: what about loading state ? seems like in the new design there's none
+   * because we just navigate to the changelog and then only we do the battery
+   * check etc. So this button would just do a plain navigation ?
```
[the remark this answers is not in the document]
Comment (ofreyssinet-ledger) [reply apps/ledger-live-mobile/src/components/FirmwareUpdateBanner.tsx:191]: oopsie will remove that comment

**Pull request merged**